### PR TITLE
[DOC] Generate simple docs for export function outputs

### PR DESCRIPTION
**Pull request by 36000**
This adds some basic documentation for the myafq.export_ functions (like myafq.export_b0()). This documentation can be seen by doing help(myafq). I will post a picture shortly of what these changes look like.

### CODE REVIEW

**Comment by 36000** [comment]
image

**Comment by 36000** [comment]
@arokem what do you think of this system for documenting the export_ functions? Developers will have to add a description of the function output(s) to the outputs variable in the task file they are updating, so it makes it convenient for developers. Is there anything else we should add to the docstrings for the export_ functions?

File: AFQ/tasks/data.py [hunk 1]
```diff
@@ -23,12 +23,28 @@
...
+    "pmap_file": """anisotropic power map file""",
+    "dti_fa_file": """DTI fractional anisotropy file""",
+    "dti_cfa_file": """DTI color fractional anisotropy file""",
+    "dti_pdd_file": """DTI PDD file""",
+    "dti_md_file": """DTI mean diffusivity file""",
+    "dki_fa_file": """DKI fractional anisotropy file""",
+    "dki_md_file": """DKI mean diffusivity file""",
+    "dki_awf_file": """DKI axonal water fraction file""",
+    "dki_mk_file": """DKI mean kurtosis file""",
+    "brain_mask_file": """brain mask file"""}
```
Comment (arokem) [inline AFQ/tasks/data.py:47]: I think we could put a little bit more information here. 

For example: `"""full path to a nifti file containing a brain mask"""`

Or: `"""full path to a nifti file containing DKI mean diffusivity"""`

**Review by arokem — COMMENTED** [review]
I think this is good. I would suggest adding just a tiny bit more information about each output. I.e., what the type is of this output (e.g., "full path to a nifti file"), see examples inline:

**Comment by arokem** [comment]
Looks like the documentation build is running out of memory in GitHub Actions?

**Comment by 36000** [comment]
I am not sure why those memory errors were happening. What do PDD and GA stand for in the context of DTI?

**Comment by 36000** [comment]
This is ready for another review/merge

**Comment by arokem** [comment]
PDD is the principal diffusion direction of the diffusion tensor. It's the eigenvector associated with the first eigenvalue, so requires no additional calculation above and beyond what FA and MD and others already require. 

GA is geodesic anisotropy. See code [here](https://github.com/dipy/dipy/blob/master/dipy/reconst/dti.py#L92-L170). Also doesn't look like it's particularly computationally demanding, except that there is an allocation of three empty volumes the XYZ size of the data on L157 - L159.

**Pull request merged**